### DISCUSSION TITLE
Does maui have a table control similar to DataGrid in wpf

### DISCUSSION THREAD

**Discussion by weiyoucheng** [dotnet/maui#11318] [Q&A]
DataGrid is the most commonly used control for data display and report in wpf, but I don't seem to see similar controls on MAUI. Is there a better tool to replace in MAUI or will it be added in the future

**Comment by xtuzy** [comment]
i find this from google: https://enisn-projects.io/docs/en/uranium/latest/themes/material/components/DataGrid

**Reply by weiyoucheng** [reply]
Thank you. I'll take a look first

**Comment by giuseppenovielli** [comment]
https://github.com/akgulebubekir/Maui.DataGrid

**Reply by weiyoucheng** [reply]
Thank you very much for your help, but it still doesn't quite meet my needs.
